webapp/src/ts/modules/contacts/contacts-content.component.ts

```diff
@@ -104,14 +104,14 @@ export class ContactsContentComponent implements OnInit, OnDestroy {
   private getUserFacility() {
     this.store.select(Selectors.getUserFacilityId)
       .pipe(first(id => id !== null))
-      .subscribe((userFacilityId) => {
-        const shouldDisplayHomePlace = userFacilityId &&
+      .subscribe((userFacilityIds) => {
+        const shouldDisplayHomePlace = userFacilityIds &&
           !this.filters?.search &&
           !this.route.snapshot.params.id &&
           !this.responsiveService.isMobile();
 
         if (shouldDisplayHomePlace) {
-          this.contactsActions.selectContact(userFacilityId);
+          this.contactsActions.selectContact(userFacilityIds[0]);
```

Comment:
Just philosophically: do we think this is the correct behavior? For all intents and purposes, this is a randomly picked facility out of the users' facilitires list.

Reply:
For the most part it is the right behavior. 
1. If the user has one `facility_id` then getting `userFacilityIds[0]` of `['id-1']` as their homeplace is fine
2. If user has several `facility_ids`, we will randomly pick one of the facilities from `['id-1', 'id-2']`. But that should be fine because both of them are `homeplaces` so we can pick either. We also have to pick one to display the details of the `homeplace` on the RHS (Contact Detail view )